### ISSUE TITLE
Playback Error

### ISSUE THREAD

**Issue by fat2slow** [YePpHa/YouTubeCenter#2262]
Lately I've been watching youtube videos and many videos don't load and just display playback error screen with playback ID

**Comment by jasonycw** [comment]
Having the same issue for me too.

Using latest stable version Chrome and YTC Developer Version - Build #538

Refreshing sometime make it works again

**Comment by fat2slow** [comment]
Ya when I refresh it works again I'm mostly noticed that it really only happens to other tabs. Like when I'm browsing my Subscriptions and I open a video in new tab when I go to it it's got the playback error it happens 9/10 and it's starting to get really annoying cause I have to refresh every tab.

**Comment by trinsic** [comment]
Yeah can confirm. When I open videos in a new tab in the background the video will fail when I go to that tab. It works with a refresh while focused on that tab.

**Comment by Trueffel** [comment]
Same. Had it happen for a few days. Now, I tried and to set prevent buffer on the autoplay. Strangely, it doesn't act like it used to which would mean you have the press the fat play icon in the video to start it. Might be a placebo, but so far it worked, but theres probably some YT stuff interfering again and i don't wanna turn YTC off cause that error seems to happen rather randomly

**Comment by fat2slow** [comment]
I think it's a flash issue not sure but everytime I go to spotify it says
sorry program requires flash to play and then it loads fine maybe theres a
flash plugin thats interfering with youtube center

**Comment by jasonycw** [comment]
Should not be a flash issue as I am using HTML5 version of YT player

**Comment by Trueffel** [comment]
Talking about flash, last night i got the no fullscreen bug which is apparently related to flash player. I looked it up and it was enabled, although i was sure to have disabled it long ago. Google enabled it again?

**Comment by adeep** [comment]
Same as here. Problem with html5 player. Chrome Beta + Tampermonkey + #538 Dev Release

**Comment by Puk4400** [comment]
Reporting same issue on Chrome Version 51.0.2704.103 m with YTC #538. 
Workaround: refreshing the page. As mentioned above, it mostly happens on new opened tabs.

**Comment by fat2slow** [comment]
So at 12 o'clock just now I noticed that the playback error is gone and I
can watch videos without having to refresh the page in other tabs.

**Comment by vanowm** [comment]
In my case only embed player doesn't work with playback error. As soon as I disable ``Enable Embeds`` it works fine.

**Comment by adeep** [comment]
@fat2slow 
Yes problem gone.
